### ISSUE TITLE
The loss of discriminator

### ISSUE THREAD

**Issue by weizequan** [ai-tor/PyTorchSRGAN#6]
Hi, for traget_real (line 179 of train file), why not use ones_const? for target_fake(line 180 of train file), why not use zeros_const?

**Comment by aitorzip** [comment]
It helps with stability, check this out: https://github.com/soumith/ganhacks
